3.5/backup-restore.md

```diff
@@ -1,11 +1,22 @@
 ---
 layout: default
-description: Backup and restore can be done via the toolsarangodump andarangorestore
+description: Backup and restore can be done via the tools arangodump and arangorestore
+title: Backup & Restore
 ---
 Backup and Restore
 ==================
 
-Backup and restore can be done via the tools
+{%- comment %}
+TODO:
+- Introduction to topic
+- Offline dumps
+- Cluster
+{% endcomment %}
+
+Regular Backups
```

Comment:
- As discussed via phone, I would like to suggest the following changes
- We should have an introduction first, before the sections “regal backups” and “hot backups”
- In particular, the section “Performing frequent backups is important…” is general and not linked to regular backups
- Also elsewhere we call the backup done with arangodump “logical” - so perhaps we can use this name instead
- Also, version in which the hotbackup is introduced is missing
- Also, we could mention “cold” / raw backups (when server is down)
- Typo “restore” —> “restored”
- Missing introduction to what a logical backup is
- I have opened PR https://github.com/arangodb/docs/pull/167

Reply:
all handled but "Missing introduction to what a logical backup is" (lower prio)